gnd/src/mock/google-services.json

```diff
@@ -0,0 +1,51 @@
+{
+  "project_info": {
+    "project_number": "",
+    "project_id": ""
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:123456789012:android:1234567890123456",
+        "android_client_info": {
+          "package_name": "gnddemo1.appspot.com"
```

Comment:
Let's create a separate project for this? I can set one up.

Reply:
That would be great!